pilot/pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -71,9 +69,13 @@ spec:
               fieldPath: metadata.name
         - name: ISTIO_META_INTERCEPTION_MODE
           value: REDIRECT
-        image: docker.io/istio/proxyv2:unittest
+        image: gcr.io/istio-release/proxyv2:master-latest-daily
```

Comment:
Why need to change this?

Reply:
We don't need it. Actually, we didn't want it. But those are the values when just executing the unit tests. No idea where the original values came from. Those values seemed to be ignored when comparing the test results anyway,